contracts/merkle-distributor/AcrossMerkleDistributor.sol

```diff
@@ -39,26 +50,47 @@ contract AcrossMerkleDistributor is MerkleDistributor {
     /**
      * @notice Batch claims to reduce gas versus individual submitting all claims. Method will fail
      *         if any individual claims within the batch would fail.
-     * @dev    All claim recipients must be equal to msg.sender or claimer must be whitelisted.
+     * @dev    All claim recipients must be equal to msg.sender.
      * @param claims array of claims to claim.
      */
     function claimMulti(Claim[] memory claims) public override {
-        if (!whitelistedClaimers[msg.sender]) {
-            uint256 claimCount = claims.length;
-            for (uint256 i = 0; i < claimCount; i++) {
-                require(claims[i].account == msg.sender, "invalid claimer");
-            }
+        uint256 claimCount = claims.length;
+        for (uint256 i = 0; i < claimCount; i++) {
```

Comment:
Undoes changes in https://github.com/across-protocol/contracts-v2/pull/195/files that allow a whitelisted claimer to claim on behalf of claimer